projects/swimlane/ngx-ui/src/lib/components/overlay/resize-overlay/resize-overlay.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, Input, ViewEncapsulation, OnInit, OnDestroy } from '@angular/core';
+import { ɵMatchMedia } from '@angular/flex-layout';
+import { Observable } from 'rxjs';
+import { map, startWith } from 'rxjs/operators';
+import { HotkeysService } from '../../hotkeys';
+
+@Component({
+  selector: 'ngx-resize-overlay',
+  templateUrl: './resize-overlay.component.html',
+  styleUrls: ['./resize-overlay.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class ResizeOverlayComponent implements OnInit, OnDestroy {
+  static COMBO = 'ctrl+shift+o';
+
+  @Input()
+  get query(): string {
+    return this._query;
+  }
+
+  set query(value: string) {
+    this._query = value;
+    this._buildObservable();
+  }
+
+  @Input()
+  set disabled(value) {
+    this._disabled = value;
+    localStorage.setItem('overlay-disabled', value.toString());
+  }
+
+  get disabled() {
+    return this._disabled;
+  }
+
+  visible$: Observable<boolean>;
+
+  private _disabled: boolean = JSON.parse(localStorage.getItem('overlay-disabled')) || false;
```

Comment:
Should wrap this in a try/catch or use `localStorage.getItem('overlay-disabled') === 'true'` which should work without the risk of `JSON.parse` thowing an error.